_pages/contributing/definition-of-done.md

```diff
@@ -113,6 +117,27 @@ The coverage report can be found in `./coverage/index.html`.
 
 The UI of the feature complies with the Accessibility success criteria that are defined for the given released version. This includes writing [accessibility end-to-end tests]({{ site.baseurl }}{% link _pages/contributing/a11y-e2e-tests.md %}). For more information, see [Accessibility Best Practices]({{ site.baseurl }}{% link _pages/dev/accessibility/best-practices/a11y-best-practices.md %}).
 
+## Browser Compatibility
+
+At minimum, manual happy path test of new feature is successful and there are no significant layout issues in the following browsers:
+- Chrome 87.0.4280
+- Firefox 86.0
+- Safari 14.0.1
+- Edge 88.0.705.63
+
+Version numbers subject to change periodically. Internet Explorer not officially supported. 
+
+## Device Compatibility
+
+New feature must be compatible with major mobile devices in the marketplace. Manual happy path test must pass with: 
+- iPhone 7, 10, X ,11 (at least one)
```

Comment:
I think it's iphone 8. 7 is too old. Also, we should focus on mobile Safari versions.

Reply:
ok. Will change to iPhone8. 
Safari mobile versions are tied to the iOS version itself afaik.

Reply:
Added that new features "must be compatible with Safari on iOS, and Chrome on Android", and also mentioned that phones and tablets "should be running on the latest versions of their respective operating systems."